src/sandstorm/grain.capnp

```diff
@@ -401,6 +401,14 @@ interface UiView {
     # two UiViews are really the same view with different permissions applied, and can then combine
     # them in the UI as appropriate.
     #
+    # Implementation-wise, in addition to the permissions enumerated here, there is an additional
+    # "is human" pseudopermission provided by the Sandstorm platform, which is invisible to apps,
+    # but affects how UiView capabilities may be used.  In particular, all users are said to possess
+    # the "is human" pseudopermission, whereas all other ApiTokenOwners do not, and by default, the
+    # privilege is not passed on.  All method calls on UiView require that the caller have the "is
```

Comment:
Well, child tokens, or their owners, in the same way that existing tokens may have a `roleAssignment` which passes on permissions to their children.  Is there another way of thinking about it that I'm missing/forgetting?